update_info.py

```diff
@@ -49,9 +55,22 @@ def get_description_url(element):
 
 for key, info in entryInfo.items():
     if key not in originalEntryInfo:
+        for k,v in info.items():
+            print(k,v)
+        available = valid_input('Make this addon available? (y/n) ')
+        info['available'] = True if available == 'y' else False
+        notes = input('Add notes on this addon: ')
+        info['notes'] = notes
         originalEntryInfo[key] = info
     else:
-        originalEntryInfo[key].update(info)
+        for k,v in info.items():
+            if originalEntryInfo[key][k] != v:
+                print(f"{key}: Old value for {k}: {originalEntryInfo[key][k]}")
+                print(f"{key}: New value for {k}: {v}")
+                decision = valid_input('Replace old value with new value? (y/n) ')
+                if decision == 'y':
+                    originalEntryInfo[key][k] = v
```

Comment:
Same comment here. It would be less controlling, as it would `y` or bust. Again, who cares?